arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,360 @@
+            self.utilityNetwork?.associations(withExtent: self.graphicsOverlay.extent) { containmentAssociations, _ in
+                guard let containmentAssociations = containmentAssociations else { return }
+                containmentAssociations.forEach { association in
+                    var symbol = AGSSymbol()
+                    if association.associationType == .attachment {
+                        symbol = self.attachmentSymbol.symbol
+                    } else {
+                        symbol = self.connectivitySymbol.symbol
+                    }
+                    self.graphicsOverlay.graphics.add(AGSGraphic(geometry: association.geometry, symbol: symbol))
+                }
+                // Enable the bar button item to exit the container view.
+                self.exitBarButtonItem.isEnabled = true
+            }
+        }
+    }
+    
+    // MARK: - AGSGeoViewTouchDelegate
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        // Turn off user interaction to avoid straying away from the container view.
+        self.mapView.isUserInteractionEnabled = false
+        // Identify the top most feature that corresponds to the tapped point.
+        mapView.identifyLayers(atScreenPoint: screenPoint, tolerance: 5, returnPopupsOnly: false) { [weak self] (layerResults, error) in
+            guard let self = self else { return }
+            if let layerResults = layerResults, !layerResults.isEmpty {
+                self.identifyContainerFeature(layerResults: layerResults)
+            } else {
+                if let error = error {
+                    self.presentAlert(error: error)
+                }
+                self.mapView.isUserInteractionEnabled = true
+            }
+        }
+    }
+    
+    // MARK: - UIViewController
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        AGSAuthenticationManager.shared().delegate = self
+        // Load the utility network.
+        loadUtilityNetwork()
+        // Add a graphics overlay.
+        mapView.graphicsOverlays.add(graphicsOverlay)
+        // Get the legends from the feature service.
+        fetchLegendInfo()
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = ["DisplayContentUtilityNetworkContainerViewController"]
+    }
+    
+    // MARK: - Navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "DisplayContentLegendSegue",
+            let controller = segue.destination as? UITableViewController {
+                controller.presentationController?.delegate = self
+                controller.tableView.dataSource = symbolsDataSource
+            }
+    }
+    
+    // MARK: - UIAdaptivePresentationControllerDelegate
```

Comment:
```suggestion
    // MARK: - UIAdaptivePresentationControllerDelegate
    
```

However, usually this type of conformance is done in an extension.